collectors/timex.plugin/README.md

```diff
@@ -6,13 +6,16 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/timex
 
 # timex.plugin
 
-This plugin monitors the system clock synchronization state on Linux nodes.
+This plugin monitors the system kernel clock synchronization state on Linux nodes.
 
 This plugin creates two charts:
 
--   System clock synchronization state
+-   System clock synchronization state according to the system kernel
 -   Computed time offset between local system and reference clock
 
+This is obtained from the information provided by the ntp_adjtime() system call.
```

Comment:
```suggestion
This is obtained from the information provided by the [ntp_adjtime()](https://man7.org/linux/man-pages/man2/adjtimex.2.html) system call.
```

Reply:
good idea, that has been my source for most of this